src/lib/radio/radio.ts

```diff
@@ -280,10 +300,26 @@ export class MdRadioButton implements OnInit {
   name: string;
 
   /** Used to set the 'aria-label' attribute on the underlying input element. */
-  @Input('aria-label') ariaLabel: string;
+  _ariaLabel: string;
+  @Input('aria-label')
+  get ariaLabel(): string {
+    return this._ariaLabel;
+  }
+  set ariaLabel(value: string) {
+    this._ariaLabel = value;
+    this._change.markForCheck();
```

Comment:
Then there's a bug for OnPush since the properties are not updated when their values changed. 

The `writeValue` function is in `MdRadioButtonGroup` which is a `directive` and has no `changeDetection` strategy.